draft-packages/popover/KaizenDraft/Popover/PopoverLegacy.tsx

```diff
@@ -0,0 +1,108 @@
+import { Icon } from "@kaizen/component-library"
+import closeIcon from "@kaizen/component-library/icons/close.icon.svg"
+
+import classNames from "classnames"
+import * as React from "react"
+
+import styles from "./styles.scss"
+import { Side, Size, Variant, Position } from "./types"
+import {
+  mapArrowPositionToClassLegacy,
+  mapArrowSideToClassLegacy,
+  mapArrowVariantToClass,
+  mapLineVariant,
+  mapSizeToClass,
+  mapVariantToBoxClass,
+  mapVariantToIcon,
+  mapVariantToIconClass,
+} from "./classMappers"
+
+export type LegacyPopoverProps = {
```

Comment:
Nothing really new in this file. It's the old `Popover.tsx` code moved over. With the following exceptions:
* The class mappers have been moved to `classMappers.ts`.
* The `automationId` prop is fixed
* The `id` prop is removed, because it wasn't getting used.
* Minor changes to how the arrow works, similar to what was done in Popover.elm